### PR TITLE
In docker

### PR DESCRIPTION
**Pull request by shizacat**
Hi,
Your docker looks weird.
Putting the keys in the docker is not correct.
Also, the container itself contains many layers (RUN), the size of the steel is two times smaller (1222Mb vs 668 Mb)

Well, for dessert, after updating the library **bokeh** it stopped rendering normally. I just fixed the version so as not to disrupt the ability. There is not enough to fix the knowledge of the library (you still have a chance ;) )

### CODE REVIEW

File: docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+WORK_PATH=/opt/service
+DATA_PATH=${WORK_PATH}/data
+
+# app setup
+if [ -z "$(ls -A ${DATA_PATH})" ]; then
+	python3 ${WORK_PATH}/setup_db.py
+fi
+
+if ! [ -z "${USE_PROXY}" ]; then
```
Comment (bkueng) [inline docker/run.sh]:
```suggestion
if [ -n "${USE_PROXY}" ]; then
```

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+RUN apt-get	update && \
+	apt-get install -y --no-install-recommends apt-utils python3 python3-pip sqlite3 fftw3 python3-setuptools && \
+	apt-get install -y --no-install-recommends python3-dev libfftw3-dev git && \
+	pip3 install --no-cache-dir -r requirements.txt && \
+	apt-get remove -y python3-dev libfftw3-dev git && \
+	apt-get -y autoremove && \
+	rm -rf /var/lib/apt/lists/*
+
+
+## set cesium and mabox keys
```
Comment (bkueng) [inline docker/Dockerfile]: Can you remove these comments?

Comment (shizacat) [reply docker/Dockerfile]: Yes, of course. Comments in bottom are corrected according to the Dockerfile.

**Comment by bkueng** [comment]
Rebased, squased the 2 last commits into the first and merged.

**Pull request closed without merging**